pkg/bloomgateway/bloomgateway.go

```diff
@@ -69,13 +69,6 @@ import (
 
 var errGatewayUnhealthy = errors.New("bloom-gateway is unhealthy in the ring")
 
-// TODO(chaudum): Make these configurable
-const (
-	numWorkers             = 4
-	maxTasksPerTenant      = 1024
-	pendingTasksInitialCap = 1024
```

Comment:
I don't think this needs to be configurable. It defines the initial capacity of the `pendingTasks` map, which is used to track the queue size.
There is not much to tweak with this value.

Reply:
Done :)